pkg/service/service.go

```diff
@@ -108,7 +108,7 @@ func List(client *occlient.Client, applicationName string, projectName string) (
 	return services, nil
 }
 
-// List lists all the deployed services and additionally resolve provide the "smart" status of each one
+// ListWithSmartStatus lists all the deployed services and additionally resolve provide the "smart" status of each one
```

Comment:
This part of the text is not really readable `additionally resolve provide the "smart" status`.